src/LAMMPS.jl

```diff
@@ -50,6 +57,7 @@ function check(lmp::LMP)
 end
 
 function command(lmp::LMP, cmd)
+    check_init(lmp)
```

Comment:
Why do we need these calls to `check_init`? We should error on constructing an incorrect LAMMPS instance instead.

Reply:
Fair enough. Will make sure the constructor enforces proper LAMMPS instances